modules/preview-react/menu/stories/Menu.stories.mdx

```diff
@@ -27,8 +26,15 @@ yarn add @workday/canvas-kit-preview-react
 
 ### Basic Example
 
+`Menu` is typically triggered by an action such as pressing a button. Here's an example of how you
+might implement that behavior using a [`Popup`](/components/popups/popup/).
+
 <ExampleCodeBlock code={Basic} />
 
```

Comment:
Discussed this @alanbsmith a bit: I removed the current `Basic` example and made `ControlButton` the new `Basic` example instead. Seemed odd to have a Menu just floating on its own without a triggering element for our main example.